mobile/src/bundles/common/components/input/input.tsx

```diff
@@ -54,7 +54,7 @@ const Input = <T extends FieldValues>({
             )}
             <TextInput
                 onChangeText={onChange}
-                value={value}
+                value={value.toString()}
```

Comment:
I added this because otherwise I was getting 
`'Warning: Failed prop type: Invalid prop "value" of type "number" supplied to "TextInput", expected "string"`
 for **Input name = "salaryExpectation"**, whose type is a **number** in TalentOnboardingProfileDto